.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/
+always-auth=true
```

Comment:
I think this line is specific to aspnet core, and not even sure it's required anymore.
Had to deal with it during the holidays. If you look at all the other dotnet repos they don't have it. It's supposed to allow azdo to authenticate to the feeds (all the upstream ones) using the tokens configured in azdo. But these are public and shouldn't require auth. That's what I understood.

Reply:
@wtgodbe

Reply:
@sebastienros not exactly. other repos don't do this b/c they don't use `npm` or `yarn`. and, auth is for updating the feed when an authorized person like you grabs packages from the upstream feed and updates lock files

Reply:
Javier told me 2 weeks ago and I already had forgotten.

So I believe we don't need that here. There is nothing that requires the feed to be updated (packages.lock file), right @eerhardt ?

Reply:
I'm just following the docs from the error: https://aka.ms/cfs/npm. See `Mitigation Steps` section.

Reply:
I merged what I had to unblock the supply chain validation. We can tweak this as we go.